publication-event-handlers/src/test/java/no/unit/nva/publication/events/handlers/delete/DeleteFileFromS3EventHandlerTest.java

```diff
@@ -0,0 +1,89 @@
+package no.unit.nva.publication.events.handlers.delete;
+
+import static no.unit.nva.model.testing.associatedartifacts.AssociatedArtifactsGenerator.randomOpenFile;
+import static no.unit.nva.publication.events.bodies.DataEntryUpdateEvent.FILE_ENTRY_DELETE_EVENT_TOPIC;
+import static no.unit.nva.publication.events.handlers.PublicationEventsConfig.EVENTS_BUCKET;
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static no.unit.nva.testutils.RandomDataGenerator.randomUri;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import com.amazonaws.services.lambda.runtime.Context;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.UUID;
+import no.unit.nva.events.models.EventReference;
+import no.unit.nva.identifiers.SortableIdentifier;
+import no.unit.nva.publication.events.bodies.DataEntryUpdateEvent;
+import no.unit.nva.publication.model.business.FileEntry;
+import no.unit.nva.publication.model.business.UserInstance;
+import no.unit.nva.s3.S3Driver;
+import no.unit.nva.stubs.FakeContext;
+import no.unit.nva.stubs.FakeS3Client;
+import no.unit.nva.testutils.EventBridgeEventBuilder;
+import nva.commons.core.Environment;
+import nva.commons.core.paths.UnixPath;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import software.amazon.awssdk.services.s3.model.NoSuchKeyException;
+
+class DeleteFileFromS3EventHandlerTest {
+
+    private static final Context CONTEXT = new FakeContext();
+    private DeleteFileFromS3EventHandler handler;
+    private FakeS3Client s3Client;
+    private ByteArrayOutputStream output;
+
+    @BeforeEach
+    public void setUp() {
+        output = new ByteArrayOutputStream();
+        s3Client = new FakeS3Client();
+        handler = new DeleteFileFromS3EventHandler(s3Client);
+    }
+
+    @Test
+    void shouldDeleteFileFromS3WhenFileEntryHasBeenDeleted() throws IOException {
```

Comment:
This test doesn't appear to work correctly. If you remove the line `handler.handleRequest(event, output, CONTEXT);`, the test still passes. This indicates to me that the expected file isn't inserted at the expected path, but could be something else.

Reply:
Thank you, seems like i forgot to persist file to S3 before deleting it :)